### DISCUSSION TITLE
Seeking Assistance: Need Help Finding Official Mastodon Logo for Beginner's First Open Source Project

### DISCUSSION THREAD

**Discussion by HarigovindM-G** [mastodon/mastodon#25139] [Q&A] [answered]
Hey Mastodon Community,

I'm a complete beginner diving into the world of open source development, and I'm super excited to make my first contribution! Currently, I'm working on a project that involves integrating Mastodon, and I'd love to include the official Mastodon logo to give it that authentic Mastodon vibe.

But here's the catch: I'm having a tough time finding the official Mastodon logo assets. I've checked the Mastodon repository and the branding page (joinmastodon.org/branding), but no luck so far. I totally get the importance of respecting licenses and branding guidelines, and I want to do things right.

So, I'm kindly reaching out to you all for some guidance and help in getting the official Mastodon logo assets. It would be awesome if you could also share any licensing information or guidelines for using the logo in external projects.

As a newbie in the open source world, including the official Mastodon logo in my project will not only make it visually appealing but also showcase my connection to the Mastodon platform.

Thanks a ton for your understanding and support. I'm eager to learn and contribute to the Mastodon community!

Cheers,
Harigovind M G

**Comment by trwnh** [comment, marked as answer]
what do you mean by "official logo assets"? https://joinmastodon.org/branding has the wordmark and logo as the first two things on the page, and they're downloadable. all the information you need should be on that page.
